Resources/Prototypes/_DeadSpace/Loadouts/role_loadouts.yml

```
@@ -4,6 +4,7 @@
   groups:
   # - GroupTankHarness # outerClothing occupied by startingGear
   - BlueShieldOfficerBackpack
+  - BlueShieldOfficerNeck
   - BlueShieldOfficerJumpsuit
```

```suggestion
  - BlueShieldOfficerJumpsuit
  - BlueShieldOfficerNeck
```

Так как в loadout_groups.yml после Jumpsuit группа